neuvector-scanner.yaml

```diff
@@ -68,6 +97,9 @@ test:
 
 update:
   enabled: true
+  schedule:
+    period: daily
+    reason: NeuVector scanner is updated daily for database updates
```

Comment:
I don't _think_ we need to build these `daily`: it looks like a new release gets tagged for every new database version, so I think we should almost always get a new version when the `daily` version would contain a change. (This also avoids having to build the trunk version of the `scanner` code, which I think would require a bit more analysis to choose to do.)